libtbx/auto_build/install_conda.py

```diff
@@ -715,7 +715,7 @@ def create_environment(self, builder='cctbx', filename=None, python=None,
       command_list.append('--copy')
     if offline and not yaml_format:
       command_list.append('--offline')
-    if builder in ("dials", "dials-old", "xfel") and command != "create":
+    if builder in ("dials", "dials-old", "xfel") and command_list[1] != "env":
```

Comment:
```suggestion
    if builder in ("dials", "dials-old", "xfel") and not yaml_format:
```
There is a `yaml_format` variable that is `True` when a `.yaml` file is found. That should be consistent with the other checks for `.yaml` files.